tools/server/server.cpp

```diff
@@ -4363,7 +4366,13 @@ int main(int argc, char ** argv) {
         res_ok(res, {{ "prompt", std::move(data.at("prompt")) }});
     };
 
-    const auto handle_models = [&params, &ctx_server, &res_ok](const httplib::Request &, httplib::Response & res) {
+    const auto handle_models = [&params, &ctx_server, &state, &res_ok](const httplib::Request &, httplib::Response & res) {
+        server_state current_state = state.load();
+        std::string model_meta;
```

Comment:
```suggestion
        json model_meta = nullptr;
```

Otherwise, the `model_meta` will always be a string, right?